### PR TITLE
(SIMP-4009) Add support for Puppet 5 and OEL

### PR DESCRIPTION
**Pull request by jeefberkey**
SIMP-4009 #comment updated simp_nfs

### CODE REVIEW

**Comment by jeefberkey** [comment]
https://gitlab.com/simp/pupmod-simp-simp_nfs/pipelines/30125311

**Comment by trevor-vaughan** [comment]
@jeefberkey GitLab tests didn't run the OEL tests (which is half of what this change was all about)

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -162,15 +169,59 @@ default-old-autofs:
     - bundle exec rake spec_clean
     - bundle exec rake beaker:suites[default]
 
+default-old-autofs-puppet5:
```
Comment (trevor-vaughan) [inline .gitlab-ci.yml]: If there's a `default_old_autofs`, I would expect to see tests that do something with new autofs.

**Comment by jeefberkey** [comment]
https://gitlab.com/simp/pupmod-simp-simp_nfs/pipelines/30210416

**Pull request merged**